### PR TITLE
Revise workflow for updating templates

### PR DESCRIPTION
**Pull request by beckyjackson**
The general workflow is similar to that for the IHCC data harmonization:

1.  Create a new branch
2. Run `make init-cogs EMAIL=user@email.com`
3. Open Google sheet (link shown by step 2) and make changes
4. Run `make update`

I also removed the template strings from all `src/ontology/templates` files so they will look a little less confusing 😄

### CODE REVIEW

**Comment by jamesaoverton** [comment]
I think that I want the ROBOT template rows in GECKO. I don't want the ROBOT template rows in the `data-harmonization` any more, since those are for curators. But GECKO is an ontology and I expect it to be edited by ontology developers. I must have been unclear about that -- sorry!

The rest of this looks good.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
Please restore the ROBOT template rows.

**Comment by beckyjackson** [comment]
Done!

File: Makefile [hunk 1]
```diff
@@ -208,6 +197,22 @@ build/report.html: gecko.owl | build/robot.jar
...
 
+### COGS Set Up
+
+BRANCH := $(shell git branch --show-current)
+
+init-cogs: .cogs
+
+# required env var GOOGLE_CREDENTIALS
+.cogs: | $(TEMPLATES)
+	cogs init -u $(EMAIL) -t "GECKO $(BRANCH)" $(foreach T,$(TEMPLATES), && cogs add $(T) -r 1)
```
Comment (jamesaoverton) [inline Makefile]: With the template rows restored, this should be `cogs add $(T) -r 3`?

Comment (beckyjackson) [reply Makefile]: Just `-r 2` because we don't have a validation row right now, but I've fixed this.

**Review by jamesaoverton — APPROVED** [review]

**Pull request merged**